styles.css

```diff
@@ -0,0 +1,89 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+@media only screen and (max-width: 800px) {
+  body {
+    background-color: lightblue;
+  }
+}
+
+nav {
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 10vh;
+  background-color: #be3144;
+}
+
+nav ul {
+  display: flex;
+  justify-content: flex-end;
+}
+
+nav ul li {
+  margin-right: 40px;
+  list-style: none;
+}
+
+nav ul li a {
+  text-decoration: none;
+  color: #fff;
+  font-size: 1.5rem;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+#welcome-section {
+  height: 100vh;
+  background-color: #313235;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#welcome-section h1 {
+  color: #fff;
+  font-size: 3rem;
+  font-family: sans-serif;
+}
+
+#projects {
+  height: 50vh;
+  background-color: #45567d;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.project-tile {
+  font-size: 1.5rem;
+  color: #fff;
+}
+
+#project a {
+  display: block;
+  text-decoration: none;
+  margin-top: 10px;
+  font-size: 1.2rem;
+  color: #fff;
+  border: 1px solid #be3144;
+  border-radius: 7px;
+  padding: 10px;
+  background: #be3144;
+}
+
+#contact {
+  width: 100vw;
+  height: 10vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #313235;
+}
```

Comment:
It would be better if width is a bit smaller

Reply:
* I suggest you  explain why you request a change - it might be obvious for you but not for someone else.